pkg/services/ngalert/store/image_test.go

```diff
@@ -90,8 +90,11 @@ func TestIntegrationGetImages(t *testing.T) {
 	assert.Len(t, mismatched, 0)
 	assert.Equal(t, []models.Image{image1}, images)
 
-	// create an image with a URL
-	image2 := models.Image{Path: "https://example.com/example.png"}
+	// create an image with a long URL (e.g signed URL)
+	image2 := models.Image{URL: "https://example.com/example.png?Signature=3r4r4r34r34r34r34dfeergef3t343t34erg443" +
```

Comment:
Is the signature here just a random string?

Reply:
Yes, c&p 🐱 typing, I can change to something that looks more like a signature if that's preferable for aesthetics

Reply:
Ok, made it a little longer, over 1000 chars, this size is already longer than common signed URLs from the 3 major cloud providers.